python/examples/ingestion_with_yaml_config/main.py

```diff
@@ -36,10 +43,26 @@
         ingestion_service = IngestionService(
             channel,
             telemetry_config,
-            overwrite_rules=True,  # Overwrite any rules created in the Sift UI that isn't in the config
             end_stream_on_error=True,  # End stream if errors occur API-side.
         )
 
+        # Create/update rules
+        named_expressions = load_named_expression_modules(
```

Comment:
As I said above, let's remove named expression modules from the examples in favor of pushing asset agnostic rules flow as much as we can.